src/models/tools/geometry/geometry-content.ts

```diff
@@ -137,17 +137,21 @@ export const GeometryContentModel = types
     // actions
     function initializeBoard(domElementID: string, onCreate?: onCreateCallback): JXG.Board | undefined {
       const changes = self.changes.map(change => JSON.parse(change));
-      let board;
+      let board: JXG.Board | undefined;
       applyChanges(domElementID, changes)
         .filter(result => result != null)
         .forEach(elt => {
           if (isBoard(elt)) {
             board = elt as JXG.Board;
+            board.suspendUpdate();
```

Comment:
Is the idea that the first element in the results list is the board?

Reply:
Yes, the board has to be created before any other changes/objects so it's generally the first change in the list.